src/null-safety/index.md

```diff
@@ -1,41 +1,288 @@
 ---
 title: Null safety
-description: Watch this space for news about null safety
-toc: false
+description: Information about Dart's upcoming null safety feature
 ---
 
-Watch this page for updates about null safety.
-In the meantime, check out these resources:
+Sound null safety is coming to the Dart language!
+When you opt into null safety,
+types in your code are non-nullable by default, meaning that
+values can’t be null _unless you say they can be._
+With null safety, your **runtime** null-dereference errors
+turn into **edit-time** analysis errors.
+Because null safety in Dart is sound,
+the compiler can optimize away internal null checks,
+enabling apps to be faster, smaller, and more reliable.
 
-* [Null-safe playground: nullsafety.dartpad.dev][nullsafety.dartpad.dev]
+You can practice using null safety in the web app
+[DartPad with Null Safety.][nullsafety.dartpad.dev]
+
+{{ site.alert.important }}
+  **Don't start migrating large projects yet.**
```

Comment:
This sounds a bit like it's OK to migrate smaller projects. We really don't want even small packages migrated and published yet.

Maybe change to something like "Don't migrate production code, don't publish any migrated code."

Reply:
Or maybe slightly more nuanced: It's totally OK to migrate code (of any size), you should just not do it to your main master code base (like you could do it to a copy, or do it and then roll back).